lib/index.js

```diff
@@ -2,20 +2,29 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 const client = require("prom-client");
 function init(opts) {
-    const { queue, interval = 60000, promClient = client } = opts;
+    const { interval = 60000, promClient = client } = opts;
     const QUEUE_NAME_LABEL = 'queue_name';
     const activeMetricName = 'jobs_active_total';
     const waitingMetricName = 'jobs_waiting_total';
     const completedMetricName = 'jobs_completed_total';
     const failedMetricName = 'jobs_failed_total';
     const delayedMetricName = 'jobs_delayed_total';
+    const durationMetricName = 'jobs_duration';
```

Comment:
Could you change metric name to include time unit (milliseconds?). See: https://prometheus.io/docs/practices/naming/#metric-names